core/src/main/java/jenkins/model/item_category/Categories.java

```diff
@@ -0,0 +1,50 @@
+package jenkins.model.item_category;
+
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+import org.kohsuke.stapler.export.Flavor;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * It is a logic representation of a set of {$Category}.
+ */
+@ExportedBean
+public class Categories implements HttpResponse, Serializable {
```

Comment:
@tfennelly AFAIK, this is:

```
@Override
public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object node) throws IOException, ServletException {
    rsp.serveExposedBean(req, this, Flavor.JSON);
}
```

Really, `Categories` and `Category` are not domain object, are representations of domain objects. This is common when you work with framework like **RESTEasy** or **Restlet**.

Maybe these classes should be in a specific package, but this is unrelated with this PR.